modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -505,15 +505,17 @@ private CompletableFuture<Boolean> onCreateZone(CreateZoneEventParameters create
             return calculateZoneAssignmentsAndCreateReplicationNodes(
                     createZoneEventParameters.causalityToken(),
                     createZoneEventParameters.catalogVersion(),
-                    createZoneEventParameters.zoneDescriptor()
+                    createZoneEventParameters.zoneDescriptor(),
+                    false
             ).thenApply(unused -> false);
         });
     }
 
     private CompletableFuture<Void> calculateZoneAssignmentsAndCreateReplicationNodes(
             long causalityToken,
             int catalogVersion,
-            CatalogZoneDescriptor zoneDescriptor
+            CatalogZoneDescriptor zoneDescriptor,
+            boolean nodeRecovery
```

Comment:
We already have similar methods with this parameter being called `boolean onNodeRecovery`. I would suggest to make them all have the same name (either your variant or the other)